python/pytests/golden/collect_test/test_collect_bool_trailing_window_1s.jsonl

```diff
@@ -1,6 +1,14 @@
 {"_time":"1996-12-19T16:39:57.000","_subsort":0,"_key_hash":12960666915911099378,"_key":"A","b":true,"collect_b":[true]}
+{"_time":"1996-12-19T16:39:58.000","_subsort":0,"_key_hash":12960666915911099378,"_key":"A","b":null,"collect_b":[]}
```

Comment:
Empty list rows like this represent the "clearing" of the window after `duration` has passed.